website/versioned_docs/version-22.3/SnapshotTesting.md

```diff
@@ -136,7 +136,38 @@ From here you can choose to update that snapshot or skip to the next:
 
 ![](/jest/img/content/interactiveSnapshotUpdate.gif)
 
-### Tests Should Be Deterministic
+## Best Practices
+
+Snapshots are a fantastic tool for identifying unexpected interface changes
+within your application – whether that interface is a UI, logs, or error
+messages; however, they're not a cure-all, and, as with any testing strategy,
+there are some best-practices you should be aware of, and guidelines you should
+follow, in order to use them effectively.
+
+### 1. Treat Snapshots As First-Class Citizens
+
+Commit snapshots and review them as part of your regular code review process.
+This implies treating snapshots as you would any other type of test, or code for
+that matter, in your project.
+
+Ensure that your snapshots are _very_ readable by keeping them focused, short,
+and using tools that enforce these stylistic conventions.
+
+As mentioned previously, Jest uses
+[pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format)
```

Comment:
Maybe link to https://www.npmjs.com/package/pretty-format instead?

Reply:
Or https://yarnpkg.com/en/package/pretty-format

Reply:
Sure 🤷‍♂️